packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx

```diff
@@ -41,28 +41,37 @@ import { TitleBar } from './TitleBar'
 import { TrackInfo } from './TrackInfo'
 import { PLAY_BAR_HEIGHT } from './constants'
 
-const combinedBottomAreaHeight = BOTTOM_BAR_HEIGHT + PLAY_BAR_HEIGHT
-
 const STATUS_BAR_FADE_CUTOFF = 0.6
 const SKIP_DURATION_SEC = 15
+// If the top screen inset is greater than this,
+// the status bar will be hidden when the drawer is open
+const INSET_STATUS_BAR_HIDE_THRESHOLD = 20
 
 const useStyles = makeStyles(({ spacing }) => ({
   container: {
     paddingTop: 0,
-    height: FULL_DRAWER_HEIGHT - combinedBottomAreaHeight
+    height: FULL_DRAWER_HEIGHT,
+    justifyContent: 'space-evenly'
+  },
+  playBarContainer: {
```

Comment:
ah, and is this so playbar is not calculated in overall drawer height?

Reply:
Pretty much, this makes it so there isn't a big blank space at the top of the drawer where the playbar would be. Important for devices without a notch